internal/debug/flags.go

```diff
@@ -74,7 +79,7 @@ var (
 // Flags holds all command-line flags required for debugging.
 var Flags = []cli.Flag{
 	verbosityFlag, vmoduleFlag, backtraceAtFlag,
-	pprofFlag, pprofPortFlag,
+	pprofFlag, pprofPortFlag, pprofAddrFlag,
```

Comment:
Please swap the addr and port flag orders. That way it will be consistent with the ordering of other flags.